### PR TITLE
CC-6 | Import Uber konnector

### PR DESCRIPTION
**Pull request by m4dz**
Imported from [konnectors v2](https://github.com/cozy-labs/konnectors/blob/master/server/konnectors/uber.js), it seems to be ok but I haven't any Uber acount to test. @doubleface any chance you can extensively test it?

### CODE REVIEW

**Review by gregorylegarec — APPROVED** [review]

File: konnector.js [hunk 1]
```diff
@@ -1,91 +1,229 @@
...
+      jar: true,
+      url: 'https://login.uber.com/login',
+      form: {
+        email: requiredFields.login,
+        password: requiredFields.password,
+        _csrf_token: token
+      }
+    }
+
+    log.info('Logging in')
```
Comment (doubleface) [inline konnector.js]: Log does not come from the printit package anymore but from [this lib](https://github.com/cozy/cozy-konnector-libs/blob/master/logger.js) now. And it does not have the same interface. This line can be replaced by `log('info', 'Logging in')`

File: konnector.js [hunk 2]
```diff
@@ -1,91 +1,229 @@
...
+
+    return request(signInOptions, (err, res) => {
+      if (err) {
+        log.error('Login failed')
```
Comment (doubleface) [inline konnector.js]: There is now a specific error code to say that the login has failed : `LOGIN_FAILED`. You can find the long list of codes [here](https://github.com/cozy/cozy-konnector-template/blob/master/docs/ERROR_CODES.md).

This code will be interpreted by cozy-collect to display a specific error message.

**Review by doubleface — CHANGES_REQUESTED** [review]
Only some problems with the logging library, the rest looks good to me even if I did not test it with a valid uber account.

**Comment by m4dz** [comment]
@doubleface done, tell me if it's better now :)

File: konnector.js [hunk 3]
```diff
@@ -97,11 +95,11 @@ function logIn (requiredFields, bills, data, next) {
       }
       return request(tripsOptions, (err, res, body) => {
         if (err) {
-          log.error('An error occured while fetching trips information')
-          log.raw(err)
```
Comment (doubleface) [inline konnector.js]: Le `log('raw', ...` ne va malheureusement pas bien fonctionner puisque c'est un `JSON.stringify` qui est fait derrière. Et dans ce cas, tu vas obtenir un `{}` à la fin. Tu peux toujours faire log('info', err.message) ou faire un console.log dans cas, qui ne va pas générer de json mais ne fera pas non-plus planter la stack et affichera la stack trace en prime.

**Review by doubleface — APPROVED** [review]
Juste le pb de log('raw') mais sinon tout m'a l'air d'être bon.

**Comment by m4dz** [comment]
I've made some changes about the `log('raw', ...)`. I also updated the manifest and the Travis config to enable auto-build, let me know if it's ok (I still havent any Uber account to test the scrapping, but this part of the v2 code remains untouched)

**Comment by gregorylegarec** [comment]
ping @doubleface :)

**Pull request merged**